src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -70,6 +81,7 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
 
   const scheduledStartDate = sovereignVenue?.start_utc_seconds;
 
+  // @debt: probably will need to be re-calculated based on minDate instead of startOfDay.Check later
```

Comment:
![image](https://user-images.githubusercontent.com/63313732/128306082-a0a094f4-8b49-4157-934a-fe0e0e057dfa.png)

looks like something got broken a bit. Both the dates are disabled although there are events. 😬

Reply:
should be good now, fixed in 365daad